### ISSUE TITLE
does not have these features: ``

### ISSUE THREAD

**Issue by niklasf** [crev-dev/cargo-crev#290]
```
$ cargo crev crate verify
Error: Package `btoi v0.4.2 ([...]/rust-btoi)` does not have these features: ``
$ cargo crev --version
cargo-crev 0.16.0
$ uname -a
Linux [...] 5.5.3-arch1-1 #1 SMP PREEMPT Tue, 11 Feb 2020 15:35:41 +0000 x86_64 GNU/Linux
$ cargo --version
cargo 1.41.0 (626f0f40e 2019-12-03)
```

Seems to happen on every crate.

**Comment by dpc** [comment]
Thanks for reporting it. And sorry it is not working. I really have no time for this project, as I can't even test stuff properly anymore. `cargo` changeg lockfiles format, which forced me to upgrade despite lack of time, which introduced this.

**Comment by dpc** [comment]
It's odd. It happens when I'm in a sub-crate of a bigger workspace, but it doesn't when I'm in the workspace itself. Can you investigate it a bit in your project?

**Comment by niklasf** [comment]
I tried with a fresh clone of bitflags, autocfg and btoi in `/tmp` (outside of any possible workspace).

```

**Comment by dpc** [comment]
I can only offer this workaround right now:

```
cargo crev verify --features default
```

**Comment by dpc** [comment]
I think it's fixed now.

**Comment by niklasf** [comment]
Perfect, thanks! Can confirm it's fixed in v0.16.1.
